win/rl/winrl.cc

```diff
@@ -256,6 +256,8 @@ NetHackRL::fill_obs(nle_obs *obs)
         obs->internal[5] = nle_seeds[0]; /* core */
         obs->internal[6] = nle_seeds[1]; /* disp */
         obs->internal[7] = u.uhunger;
+        obs->internal[8] =
```

Comment:
We add u.urexp here, but we;re not using it anywhere?

Reply:
_If_ we at some point decided to use NetHack's "end of game" score, we would get it from here. See the commit message here https://github.com/facebookresearch/nle/pull/160/commits/c5f077c8cddb748ac9b25c5163a48f2a3987f590 for details.